coreid-nextjs-typescript/.env.example

```
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_COREID_URL=https://coreid-test.assently.com/embed/coreid.js
+ASSENTLY_SECRET=<your provided secret key from assently>
+ASSENTLY_IDENTITY_KEY=<your provided identity key from assently>
+ASSENTLY_ISSUER=<your provided issuer from assently>
+ASSENTLY_DOMAIN_NAME="Test av Mobilt BankID" # this is required for testing
```

I believe it would be more descriptive if the naming is similar to what it is already in use in our documentation. 
ref https://developer.assently.com/docs/getting-started-quick-start-guide#0-credentials-and-configuration
```suggestion
ASSENTLY_CUSTOM_DISPLAY_NAME="Test av Mobilt BankID" # this is required for testing of Swedish BankID
```